appengine_flexible/analytics/analytics.go

```diff
@@ -3,6 +3,7 @@
 // license that can be found in the LICENSE file.
 
 // Sample analytics demonstrates Google Analytics calls from App Engine flexible environment.
+// [START gae_flex_analytics_track_event]
```

Comment:
Move this comment to before the line above, and leave a newline in between (otherwise this shows up in the automatically generated godoc):

```
// [START ...]

// Sample analytics ...
package main
```

Reply:
I see this in most of the files. Can you put them all into this format? Thanks :)